tests/lib/snaps/fuse-consumer/hello.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+#    Copyright (C) 2006  Andrew Straw  <strawman@astraw.com>
+#
+#    This program can be distributed under the terms of the GNU LGPL.
+#    See the file COPYING.
+#
+
+import os, stat, errno
+# pull in some spaghetti to make this stuff work without fuse-py being installed
```

Comment:
Why do we need that?  We know exactly what lives where in this context, and the guesses made in that file don't look sensible for the snap environment.